### PR TITLE
Add Medium to social links

### PR DESCRIPTION
**Pull request by elliott-king**
Medium is a centralized writing hub.

Note that the username link is formatted `https://medium.com/@{username}` (line 17 in social.html). I felt that the readme and config should make it clear how the user should format theirs.

### CODE REVIEW

File: _includes/social.html
Comment (MichaelCurrin) [inline _includes/social.html:19]: Consider adding rel="me" for consistency with the others.

**Review by MichaelCurrin — CHANGES_REQUESTED** [review]

File: _includes/social.html
Comment (elliott-king) [reply _includes/social.html]: Gotcha, done.

If anyone comes upon this and wonders `rel="me"` does, [here](https://github.com/jekyll/minima/pull/386) is the original pull request.

Comment (MichaelCurrin) [reply _includes/social.html]: Great, thanks for sharing.

File: assets/minima-social-icons.svg
Comment (MichaelCurrin) [inline assets/minima-social-icons.svg:52]: Is `currentColor` needed here? The other icons don't have it and maybe the `fill-rule` works fine.

Comment (elliott-king) [reply assets/minima-social-icons.svg]: This reflects my lack of understanding of SVGs (I just copy-pasted it from another site). I believe you are correct. I  tested this by:
- trying all of the default color themes (dark, solarized)
- toggling for each theme

It looks like it should be covered by an [existing CSS rule](https://github.com/jekyll/minima/blob/0ebe102cd37c914305f74c6a138e26f777d19c3b/_sass/minima/_base.scss).

Removed.

Comment (MichaelCurrin) [reply assets/minima-social-icons.svg]: Thanks!

Oh yes currentColor is set on the CSS rather than the SVG itself and that works.

I use currentColor on the SVG itself in my project, which I wrote about here. I also added a section based on on Minima works.

https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/web/svg.html#current-color

File: README.md
Comment (MichaelCurrin) [inline README.md:272]: BTW there is Jekyll profile but it is unrelated to the Jekyll we know

https://medium.com/@jekyll

**Review by MichaelCurrin — APPROVED** [review]

File: README.md
Comment (elliott-king) [reply README.md:272]: Yeah, there is no official Jekyll Medium account (I think). If I go to https://micro.blog/jekyll, it is a non-existent member. If I go to https://keybase.io/team/jekyll, it is privatized. 

I figured the precedent is to use it as a placeholder.

File: assets/minima-social-icons.svg
Comment (elliott-king) [reply assets/minima-social-icons.svg]: Very nice, thanks!

File: README.md
Comment (MichaelCurrin) [reply README.md:272]: I think that is fine. It is to demonstrate

**Comment by elliott-king** [comment]
@MichaelCurrin Can I ask you to re-approve (since I rebased the branch?
Or do you not have write access?

**Review by MichaelCurrin — APPROVED** [review]

**Comment by MichaelCurrin** [comment]
I have. You have to update with `master` again though.

```
This branch is out-of-date with the base branch
```

**Comment by elliott-king** [comment]
> I have. You have to update with `master` again though.
> 
> ```
> This branch is out-of-date with the base branch
> ```

Done. Hmm, I guess I did that incorrectly, by rebasing - now it needs your approval again. Merging creates a merge commit, which I assume would also need your approval. I will make sure to do it with the Github UI next time, I guess that is the only way to do it without requiring re-approval.

**Comment by MichaelCurrin** [comment]
And yes I don't have write access so my review doesn't count towards the requirement.

**Comment by ashmaroli** [comment]
Thanks, but we won't be adding any more social media icons to this theme.

**Pull request closed without merging**
